### PR TITLE
Squash migrations

### PR DESCRIPTION
**Pull request by richard-dds**
Removed all the old alembic migrations, and created a new one which recreates the current state of the db. Added an idempotent `seed_roles` script to keep roles up to date.

This should be passing once we call the seed scripts from scriptz.

Now we know not to put models in our migrations.

### CODE REVIEW

**Comment by richard-dds** [comment]
@dandds The UUID thing is already taken care of in this PR :)

I do agree that it would be a good idea to do a sweep of all the tables and fields. To your list I'd also like to add:

- timestamps everywhere

**Comment by dandds** [comment]
@richard-dds Are all your commits here? I still see regular IDs in task order: https://github.com/dod-ccpo/atst/blob/squash-migrations/atst/models/task_order.py#L24

But yeah, timestamps everywhere would be great.

**Comment by richard-dds** [comment]
@dandds You're right, I did miss some. I just pushed some more commits that should take care of a lot of the cleanup: table names, timestamps (except on pe_numbers) and ids.

File: script/seed_roles.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
+# Add root project dir to the python path
```
Comment (patricksmithdds) [inline script/seed_roles.py]: Now that we have another seed script, the existing `script/seed.py` is a bit ambiguous -- can we rename that one to indicate that it seeds testing data?

File: atst/models/task_order.py [hunk 2]
```diff
@@ -18,13 +18,13 @@ class FundingType(Enum):
...
+class TaskOrder(Base, mixins.TimestampsMixin):
+    __tablename__ = "task_orders"
 
-    id = Column(Integer, primary_key=True)
+    id = types.Id()
     number = Column(String, unique=True)
-    source = Column(SQLAEnum(Source))
-    funding_type = Column(SQLAEnum(FundingType))
+    source = Column(SQLAEnum(Source, native_enum=False))
+    funding_type = Column(SQLAEnum(FundingType, native_enum=False))
```
[the remark this answers is not in the document]
Comment (richard-dds) [reply atst/models/task_order.py]: It prevents SQLAlchemy from creating a new Enum type in the database. Instead we're just using a string type for the column, and SQLAlchemy handles the validation.

**Comment by richard-dds** [comment]
@patricksmithdds I did do a small sweep, but I'd feel better if we all did a sweep, since there are a lot of columns here. (@dandds )

**Comment by richard-dds** [comment]
@patricksmithdds Table names are now consistent, all ids are uuids, and I think every necessary column has a NOT NULL constraint.

File: tests/domain/test_workspace_users.py [hunk 3]
```diff
@@ -26,5 +29,8 @@ def test_can_update_existing_workspace_user():
         workspace.id, [{"id": new_user.id, "workspace_role": "developer"}]
     )
 
-    assert workspace_users[0].user.id == new_user.id
-    assert workspace_users[0].workspace_role.role.name == "developer"
+    assert workspace_users[0].user.atat_role.name == new_user.atat_role.name
+    assert (
+        workspace_users[0].workspace_role.role.name
+        == new_user.workspace_roles[0].role.name
```
[the remark this answers is not in the document]
Comment (richard-dds) [reply tests/domain/test_workspace_users.py]: I believe so, but now the tests are more brittle. :(

**Pull request merged**